src/pages/studyView/addChartButton/AddChartButton.tsx

```diff
@@ -578,6 +622,39 @@ class AddChartTabs extends React.Component<IAddChartTabsProps, {}> {
 
     @observable private savingCustomData = false;
 
+    @computed private get addChartButton() {
```

Comment:
This seems specific to x vs x button.  Should method name reflect that?

Reply:
oops, not sure how that happened